pynetdicom/events.py

```diff
@@ -420,6 +420,19 @@ def dataset(self):
         )
         return self._get_dataset("DataSet", msg)
 
+    @property
+    def dataset_path(self):
+        try:
+            dataset_path = self.request._dataset_path
+        except AttributeError:
+            msg = (
+                "The corresponding event is either not a C-STORE request or "
+                "'STORE_RECV_CHUNKED_DATASET' is not set."
+            )
+            raise AttributeError(msg)
```

Comment:
thought: I would usually write this as `raise AttributeError(msg) from e` but I didn't want to break convention here

Reply:
Huh, I didn't know you could do that